kyverno.tf

```diff
@@ -58,6 +72,39 @@ resource "helm_release" "kyverno" {
   ]
   namespace = local.kyverno["create_ns"] ? kubernetes_namespace.kyverno.*.metadata.0.name[count.index] : local.kyverno["namespace"]
 
+  depends_on = [
+    helm_release.kube-prometheus-stack,
```

Comment:
I don't think dependencies are needed here

Reply:
Well isn't the `helm_release.kyverno-crds` required here ?
I probably c/c'd the `helm_rease.kube-prometheus-stack` from another addon